pheweb/serve/templates/index_react.html

```diff
@@ -117,8 +130,10 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-	    <a href="/">
+	    <span ondblclick="new Audio(window.sounds[Math.floor(Math.random() * window.sounds.length)]).play()">
 	      {{ config.logo|safe }}
+	    </span>
+	    <a href="/">
```

Comment:
Very good interactive content

Reply:
Yea I think on all error messages we should give "hommatkaa huoltomiehet noihin saatanan vehkeisiin" :D This is great...  Also I like how everything piles up on the logo.  Upsidedown project!